spec/controllers/new_administrateur/procedures_controller_spec.rb

```diff
@@ -724,4 +724,17 @@ def update_monavis
       it { expect(procedure.allow_expert_review).to be_truthy }
     end
   end
+
+  describe "GET #api_particulier" do
+    let!(:procedure) { create :procedure, :with_service, administrateur: admin }
```

Comment:
`let!` c'est plutôt un code-smell, et ici ça ne semble pas être nécessaire.

```suggestion
    let(:procedure) { create :procedure, :with_service, administrateur: admin }
```